config/certmanager/certificate.yaml

```diff
@@ -0,0 +1,26 @@
+# The following manifests contain a self-signed issuer CR and a certificate CR.
+# More document can be found at https://docs.cert-manager.io
+# WARNING: Targets CertManager 0.11 check https://docs.cert-manager.io/en/latest/tasks/upgrading/index.html for 
+# breaking changes
+apiVersion: cert-manager.io/v1alpha2
```

Comment:
can you explain what this certmanager does?

Reply:
The certmanager is used for provisioning the certificates for the webhook server.  The directory is auto-generated by operator-sdk, but it is not used since we don't enable webhook in es.

https://book.kubebuilder.io/cronjob-tutorial/cert-manager.html